src/webui/www/private/index.html

```diff
@@ -1,8 +1,8 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en" dir="ltr">
+<!DOCTYPE html>
+<html lang="${LANG}">
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=10; IE=9; IE=8;" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
This breaks the interface in IE11, please revert.
see: [screenshot](https://user-images.githubusercontent.com/9395168/32325259-372b403c-c009-11e7-8a5c-d9a8a7747713.png)